drivers/CoreVideo/include/CoreVideo.hpp

```diff
@@ -50,4 +56,28 @@ class CoreVideo
 	interface::JPEGBase &_corejpeg;
 };
 
+#define CoreVideo_IRQHandlers(instance)                                                                                \
```

Comment:
comme c'est une macro, je mettrai en uppercase

```suggestion
#define CORE_VIDEO_DECLARE_IRQ_HANDLER(instance)                                                                                
\                                                                             
```

et je me demande si on le mettrait pas dans le ST HAL? ou si on lui mettrait pas un prefix plus ST HAL, genre

```cpp
#define ST_HAL_VIDEO_DECLAIRE_IRQ_HANDLERS(instance)
// ou
#define HAL_VIDEO_DECLAIRE_IRQ_HANDLERS(instance)
// ou simplement
#define VIDEO_DECLAIRE_IRQ_HANDLERS(instance)
```